builtin/providers/aws/provider.go

```diff
@@ -215,6 +215,9 @@ func Provider() terraform.ResourceProvider {
 			"aws_lambda_permission":                        resourceAwsLambdaPermission(),
 			"aws_launch_configuration":                     resourceAwsLaunchConfiguration(),
 			"aws_lb_cookie_stickiness_policy":              resourceAwsLBCookieStickinessPolicy(),
+			"aws_elb_load_balancer_policy":                 resourceAwsLoadBalancerPolicy(),
```

Comment:
Hi @ewdurbin 

The reason i thought about renaming is they had elb then load balancer in the name - just felt very verbose

P.